testing/dataloader/csvdataloader_test.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package dataloader
+package _dataloader
```

Comment:
We could have left the files in `core/dataloader`, but it's okay to move them to the `testing` folder too. I was mainly referring to the package rename.